modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -6,54 +6,53 @@
+Programming the Intel vRAN Dedicated Accelerator ACC100 exposes the Single Root I/O Virtualization (SRIOV) virtual function (VF) devices that are then used to accelerate the FEC in the vRAN workload.
+The Intel vRAN Dedicated Accelerator ACC100 accelerates 4G and 5G Virtualized Radio Access Networks (vRAN) workloads. This in turn increases the overall compute capacity of a commercial, off-the-shelf platform.
 This device is also known as Mount Bryce.
 
-The OpenNESS SR-IOV Operator for Wireless FEC Accelerators handles the management of the Forward Error Correction (FEC) devices used to accelerate the FEC process in vRAN L1 applications.
+The SR-IOV-FEC Operator handles the management of the forward error correction (FEC) devices that are used to accelerate the FEC process in vRAN L1 applications.
 
-Configuring the SR-IOV FEC Operator involves:
+Configuring the SR-IOV-FEC Operator involves:
 
-* Creating the desired virtual functions (VFs) for the FEC device
+* Creating the virtual functions (VFs) for the FEC device
 * Binding the VFs to the appropriate drivers
 * Configuring the VF queues for desired functionality in a 4G or 5G deployment
-+
-[NOTE]
-====
-The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
-Without FEC, a garbled message would have to be resent, adding to the network load and impacting both throughput and latency.
-====
+
+The role of forward error correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
+Without FEC, a garbled message would have to be resent, adding to the network load and impacting throughput and latency.
 
 .Prerequisites
 
-* Intel® FPGA ACC100 5G/4G card
-* Node or nodes installed with the OpenNESS SR-IOV Operator for Wireless FEC Accelerators
+* Intel FPGA ACC100 5G/4G card
+* Node or nodes installed with the OpenNESS Operator for Wireless FEC Accelerators
 * Enable global SR-IOV and VT-d settings in the BIOS for the node
-* RT kernel configured with the Performance Addon Operator
+* RT kernel configured with Performance Addon Operator
 * Log in as a user with `cluster-admin` privileges
-+
-[NOTE]
-====
-All the commands run in the `vran-acceleration-operators` namespace. There is no requirement to specify the namespace as part of the `oc` commands if the user runs the `oc project vran-acceleration-operators` as described in the section Installing the SR-IOV FEC Operator.
-====
 
 .Procedure
 
-. Verify that the SR-IOV FEC Operator is installed and that pods are running:
+. Change to the `vran-acceleration-operators` project:
```

Comment:
Suggestion:

Would "Open the `vran-acceleration-operators` project:" make sense?

Reply:
@bmcelvee it looks like change is the term that is preferred in the docs.